nextflow.config

```diff
@@ -9,15 +9,16 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
+    aligner                    = null
```

Comment:
So no aligner by default, but it's a required params

Reply:
we should have a default 👍